controllers/halayerset_controller.go

```diff
@@ -20,7 +20,7 @@ import (
 	"context"
 	"fmt"
 	"github.com/go-logr/logr"
-	appv1alpha1 "github.com/mshitrit/hasno-setup-operator/api/v1alpha1"
+	appv1alpha1 "github.com/medik8s/ha-sno/api/v1alpha1"
```

Comment:
Not directly related, but aliasing api/v1alpha1 to appv1alpha looks strange (api vs app). Same at other places. i leave it up to you if you want to change